.github/workflows/size-limit.yml

```diff
@@ -1,10 +1,6 @@
 name: Size Testing
 
-on:
-  pull_request_target:
-    types:
-      - opened
-      - synchronize
```

Comment:
Why have you removed these types?

Reply:
Just updated it to the same trigger as in the `main-suite` workflow. Probably it should be enough, also it will run on each PR update